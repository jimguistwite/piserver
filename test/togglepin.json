{"synchronized":true,
 "ops":[
   {"op":"setpin", "pin":"relay1", "high":true},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay1", "high":false},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay2", "high":true},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay2", "high":false},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay3", "high":true},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay3", "high":false},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay4", "high":true},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay4", "high":false},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay5", "high":true},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay5", "high":false},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay6", "high":true},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay6", "high":false},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay7", "high":true},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay7", "high":false},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay8", "high":true},
   {"op":"wait", "duration":500},
   {"op":"setpin", "pin":"relay8", "high":false}
 ]
}
