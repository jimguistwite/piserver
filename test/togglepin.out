{"synchronized":true,
 "ops":[
   {"op":"setpin", "pin":"relay8", "high":true},
   {"op":"wait", "duration":1000},
   {"op":"setpin", "pin":"relay8", "high":false}
 ]
}
